### PR TITLE
Assignment11_JagrutiM

### PR DESCRIPTION
**Pull request by MahajanJagruti**
Assignment11_JagrutiM

### CODE REVIEW

File: src/jagruti/Assignment11_JagrutiM.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package jagruti;
+
+public class Assignment11_JagrutiM
+{
+	//a method to compute simple interest   
+	void CalculateSimpleInterest(int P,int T,int R)
```
Comment (KrishnaKTechnocredits) [inline src/jagruti/Assignment11_JagrutiM.java:6]: method name should be in small & variable too

File: src/jagruti/Assignment11_JagrutiM.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	{		
+		double SI;//Simple Interest;
+		SI = (P * T * R) / 100;
+		System.out.println("Simple Interest : "+SI);		
+	}
+	//method to convert temperature from Fahrenheit to Celsius degree.
+	void ConvertTemp(double fahrenheit)
```
Comment (KrishnaKTechnocredits) [inline src/jagruti/Assignment11_JagrutiM.java:13]: same as above

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me

File: src/jagruti/Assignment11_JagrutiM.java (hunk 1 above)
Comment (MahajanJagruti) [reply src/jagruti/Assignment11_JagrutiM.java:6]: Assignment11_Jagruti

File: src/jagruti/Assignment11_JagrutiM.java (hunk 2 above)
Comment (MahajanJagruti) [reply src/jagruti/Assignment11_JagrutiM.java:13]: Assignment11_Jagruti

**Pull request merged**